src/sidebar/components/Annotation/AnnotationHeader.js

```diff
@@ -61,6 +63,24 @@ export default function AnnotationHeader({
   const showReplyButton = replyCount > 0 && isCollapsedReply;
   const showExtendedInfo = !isReply(annotation);
 
+  // Pull together some document metadata related to this annotation
+  const documentInfo = domainAndTitle(annotation);
+  // There are some cases at present in which linking directly to an
+  // annotation's document is not immediately feasible—e.g in an LMS context
+  // where the original document might not be available outside of an
+  // assignment, and/or wouldn't be able to present associated annotations.
```

Comment:
```suggestion
  // where the original document might not be available outside of an
  // assignment (eg. Canvas files), and/or wouldn't be able to present associated annotations.
```

Just to be more concrete.